src/content/components/TreeView.js

```diff
@@ -91,35 +102,40 @@ class TreeViewRowFixedColumns extends PureComponent {
   }
 }
 
-TreeViewRowFixedColumns.propTypes = {
-  node: PropTypes.object.isRequired,
-  nodeId: PropTypes.number.isRequired,
-  columns: PropTypes.arrayOf(PropTypes.shape({
-    propName: PropTypes.string.isRequired,
-    title: PropTypes.string.isRequired,
-  })).isRequired,
-  index: PropTypes.number.isRequired,
-  selected: PropTypes.bool.isRequired,
-  onClick: PropTypes.func.isRequired,
-  highlightString: PropTypes.string,
+type TreeViewRowScrolledColumnsProps = {
+  node: Node,
+  nodeId: IndexIntoFuncStackTable,
+  depth: number,
+  mainColumn: Column,
+  appendageColumn: Column,
+  appendageButtons: string[],
+  index: number,
+  canBeExpanded: boolean,
+  isExpanded: boolean,
+  selected: boolean,
+  onToggle: (IndexIntoFuncStackTable, boolean, boolean) => any,
+  onClick: (IndexIntoFuncStackTable, MouseEvent) => any,
+  onAppendageButtonClick: ((IndexIntoFuncStackTable | null, string) => any) | null,
```

Comment:
I'll change to `mixed`; because we don't really care about their result here I don't want to restrict the functions that can be passed as prop.